api/v1alpha1/ingressnodefirewall_types.go

```diff
@@ -154,8 +154,14 @@ type IngressNodeFirewallSpec struct {
 }
 
 // IngressNodeFirewallStatus defines the observed state of IngressNodeFirewall.
-type IngressNodeFirewallStatus struct {
-}
+type IngressNodeFirewallStatus string
```

Comment:
I'd prefer to embed this under the struct. It may seem redundant now, but the way how all (?) status stuff is dealth with in k8s is that you have
~~~
status:
  field: message
~~~

I may be wrong, but I've never seen:
~~~
status: message
~~~